via/static/styles/video_player.css

```diff
@@ -3,15 +3,26 @@ CSS entry point for the video player app.
 */
 
 @tailwind base;
-
-body {
-  @apply bg-grey-0;
-  @apply font-sans text-sm text-color-text;
-}
-
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
```

Comment:
This follows Tailwind's recommendation for extending base styles: https://tailwindcss.com/docs/preflight#extending-preflight. Is there a reason we shouldn't do that?